caPutLogApp/caPutJsonLogTask.h

```diff
@@ -90,10 +93,11 @@ class epicsShareClass CaPutJsonLogTask {
      *
      * @param address IP address or hostname of the log server. Can include a port number after a colon,
      *           if port number is not specified, default value will be used.
-     * @param config Configuration paramteter. Valid value are -1 <= config <= 2.
+     * @param config Configuration parameter. Valid value are -1 <= config <= 2.
+     * @param timeout Burst filter timeout parameter.
      * @return caPutJsonLogStatus Status code.
      */
-    caPutJsonLogStatus initialize(const char* address, caPutJsonLogConfig config);
+    caPutJsonLogStatus initialize(const char* address, caPutJsonLogConfig config, double timeout);
```

Comment:
Given that there is a default timeout value (and I agree with @simon-ess about not having two definitions for it), it would be nicer to not change the API with the added parameter, so I think this method should be declared with `double timeout = DEFAULT_BURST_TIMEOUT`. The same applies to the `reconfigure()` method below, but the `setBurstTimeout()` method should not allow the parameter to be omitted.

Reply:
Agree, changed in latest commit!